condacolab.py

```diff
@@ -186,6 +297,36 @@ def install_from_url(
         "pip_task.log"
         )
 
+    print("📌 Adjusting configuration...")
+    cuda_version = ".".join(os.environ.get("CUDA_VERSION", "*.*.*").split(".")[:2])
+    prefix = Path(prefix)
+    condameta = prefix / "conda-meta"
+    condameta.mkdir(parents=True, exist_ok=True)
+
+
+    with open(condameta / "pinned", "a") as f:
+        f.write(f"cudatoolkit {cuda_version}.*\n")
+
+    with open(prefix / ".condarc", "a") as f:
+        f.write("always_yes: true\n")
+
+    if environment_file and not specs and not channels and not pip_args and not python_version:
+        extra_conda_args = extra_conda_args or ()
+        _run_subprocess(
+            [f"{prefix}/bin/python", "-m", "conda_env", "update", "-n", "base", "-f", environment_file],
+            "environment_file_update.log",
+        )
+    else:
+        _update_environment(
+            prefix=prefix,
+            environment_file=environment_file,
+            specs=specs,
+            channels=channels,
+            python_version=python_version,
+            pip_args=pip_args,
+            extra_conda_args=extra_conda_args,
+            )
+
```

Comment:
We don't need to have two separate blocks for this part, since `_update_environment()` should be able to handle empty variables just fine (i.e. only `environment_file` was passed).